src/app/pages/home/components/home.layout.tsx

```diff
@@ -8,7 +8,13 @@ import { AccountInfoCard } from './account-info-card';
 type HomeLayoutProps = Record<'currentAccount' | 'children', React.ReactNode>;
 export function HomeLayout({ children }: HomeLayoutProps) {
   return (
-    <Stack alignItems="center" width="100%" mx={['', 'space.04']}>
+    <Stack
+      alignItems="center"
+      width="100%"
+      mx={['', 'space.04']}
+      animation="fadein"
+      animationDuration="500ms"
```

Comment:
I feel like animation duration won't work here. to avoid this jumpy UI we prob need to show some loader/splash screen until page is full rendered https://stackoverflow.com/questions/40987309/react-display-loading-screen-while-dom-is-rendering